cli_reference/openshift_developer_cli/installing-odo.adoc

```diff
@@ -5,14 +5,15 @@ include::modules/common-attributes.adoc[]
 :context: installing-odo
 toc::[]
 
-The following section describes how to install `{odo-title}` on different platforms.
+The following section describes how to install `{odo-title}` on different platforms using the CLI.
+
+Currently, `{odo-title}` does not support installation in a restricted network environment.
```

Comment:
It's up to you, but to me, it feels like this warning about not being able to use odo in a restricted env belongs in the NOTE. And then the additional info about finding the binaries in the web console belongs in normal text below the NOTE.